app/controllers/api/encryptables_controller.rb

```diff
@@ -17,6 +19,8 @@ def index(options = {})
   # GET /api/encryptables/:id
   def show
     authorize encryptable
+    encryptable.paper_trail_event = 'viewed'
+    encryptable.touch
```

Comment:
Use `encryptable.paper_trail.save_with_version` here.
This will save a version without updating the encryptable's `updated_at`.
Updating the model's `updated_at` every time might have undesirable side effect such as models appearing higher in lists that sort by `updated_at` while not actually having been updated.

Reply:
Unfortunately, save_with_version does not quite work in the same way as .touch so we have to manually change the event and timestamp of the version like this
```
v = encryptable.versions.last 
v.event = 'viewed'
v.created_at = DateTime.now
v.save
```

which does not look great but it works

Reply:
hmm yes that's a bit clunky.
However a few simple things to make it more okay-ish. Maybe we'll get to a better solution later but for now: 
```
encryptable.paper_trail_event = 'viewed' # we can set that beforehand
v = encryptable.paper_trail.save_with_version # as the method returns the version object
v.created_at = DateTime.now
v.save
```
That still isn't really pretty but it atleast doesn't run the risk of getting the wrong event.